website/docs/reference/data-test-configs.md

```diff
@@ -275,3 +275,24 @@ tests:
 ```
 
 </File>
+
+#### Specify custom configurations for generic data tests
+
+_Currently available in dbt Cloud only. Specifying custom configurations for data tests will become available in dbt Core later this year._
```

Comment:
Is "later this year" v1.9? If so we can list that as a pre-req. It's in the dropdown